src/app/routes/applications/governance-list/governance-list.component.html

```diff
@@ -95,6 +95,12 @@
             <mat-icon class="btn-color-primary">preview</mat-icon>
           </button>
 
+          <!-- Create Sub-Organization -->
+          <button mat-icon-button matTooltip="Create Sub-Organization" (click)="newSubOrg(element)"
+            *ngIf="listType === ListType.ORG">
+            <mat-icon class="btn-color-primary btn-action-3" svgIcon="add-application-icon"></mat-icon>
```

Comment:
Is this as temporary icon ?

Reply:
Yes, it is temporary for now. Davy will create an icon for this.